src/index.ts

```diff
@@ -0,0 +1,61 @@
+import express from 'express'
+import promBundle from 'express-prom-bundle'
+import { Metrics } from '@eth-optimism/common-ts'
+import * as dotenv from 'dotenv'
+
+import { ReplicaMetrics } from './types'
+import { runSyncCheck } from './sync-check'
+
+
+/**
+ * Helper functions
+ */
+
+const initMetrics = (app: express.Express): ReplicaMetrics => {
+  const metrics = new Metrics({
+    prefix: 'replica_health',
+    labels: {
+      network: process.env.PROJECT_NETWORK,
+      gethRelease: process.env.L2GETH_IMAGE_TAG,
+    },
+  })
+  const metricsMiddleware = promBundle({
+    includeMethod: true,
+    includePath: true,
+  })
+  app.use(metricsMiddleware)
+
+  return {
+    lastMatchingStateRootHeight: new metrics.client.Gauge({
+      name: 'last_matching_state_root_height',
```

Comment:
Let's ADD a common prefix to these metrics to make them easier to find.

replica_health_last_matching_state_root_height
replica_health_height
replica_health_sequencer_height